src/actions/ui.js

```diff
@@ -10,17 +10,22 @@ import type { SymbolSearchType } from "../reducers/ui";
 
 export function toggleProjectSearch(toggleValue?: boolean) {
   return ({ dispatch, getState }: ThunkArgs) => {
-    if (toggleValue != null) {
-      dispatch({
-        type: constants.TOGGLE_PROJECT_SEARCH,
-        value: toggleValue
-      });
-    } else {
-      dispatch({
+    const projectSearchState = getProjectSearchState(getState());
+    if (toggleValue === null) {
```

Comment:
my bet is that this should be `undefined`.

Here's the failing test. https://github.com/jasonLaster/debugger.html/blob/preview-story/src/actions/tests/ui.js#L15-L22

You can run it with jest. `jest src/actions/` and make changes to debug it :)